pkg/setting/setting.go

```diff
@@ -1782,6 +1783,7 @@ func readGRPCServerSettings(cfg *Cfg, iniFile *ini.File) error {
 	certFile := server.Key("cert_file").String()
 	keyFile := server.Key("cert_key").String()
 	if useTLS {
+		// TODO LND Review this for GRPC?
```

Comment:
don't forget to remove or address this TODO

Reply:
Actually, I think the password should be used here, right?

I am wondering if the loading of the cert should be done in `setting.go`, and we can store the certificate directly on the setting struct and avoid having to use the tls library in other places in the code.

Reply:
ouch, thanks for pointing that, I dont think that one is actually needed. There are different kind of configs, and is not used that much for what I understand. I will assess this in the comment of the issue